ssl/statem/statem_clnt.c

```diff
@@ -904,6 +904,14 @@ int ossl_statem_client_construct_message(SSL *s, WPACKET *pkt,
         break;
 
     case TLS_ST_CW_END_OF_EARLY_DATA:
+#ifndef OPENSSL_NO_QUIC
+        /* QUIC does not send EndOfEarlyData, draft-ietf-quic-tls-24 S8.3 */
```

Comment:
nit: any comments around drafts should probably reflect RFC status now.

Reply:
Yup, we can do that on a clean up commit

Reply:
(The man pages also have some draft references, too.)